src/osd.c

```diff
@@ -364,6 +378,25 @@ render_osd(struct server *server, cairo_t *cairo, int w, int h,
 			case LAB_FIELD_TYPE:
 				buf_add(&buf, get_type(*view));
 				break;
+			case LAB_FIELD_WINFO:
+				if (rc.window_switcher.all_workspaces) {
+					buf_add(&buf, (*view)->workspace->name);
+					if ((*view)->maximized) {
+						buf_add(&buf, " M ");
+					} else if ((*view)->minimized) {
+						buf_add(&buf, " m ");
+					} else if ((*view)->fullscreen) {
+						buf_add(&buf, " F ");
+					} else {
+						buf_add(&buf, "   ");
+					}
+				}
+				buf_add(&buf, get_winfo(*view));
+				if (rc.window_switcher.all_workspaces &&
+						wl_list_length(&server->outputs) > 1) {
+					buf_add(&buf, (*view)->output->wlr_output->name);
+				}
```

Comment:
I am not sure right now if `view->output` is always set, likely it is not. So we need an additional `&& output_is_usable(*view->output)` in the condition.

Reply:
All I can say is that the data was never null or wrong when I tested, but I don't use both monitors
that much. And I would argue that if the data is incorrect, then it needs to be fixed to be accurate.

I would say leave it as is, and IF there are reports from those with multi-monitor setups, then 
address this. I'm always open to fixing what I write. ~lol~

Reply:
I thought about it, and made the change, they seem to work alright.
Whether more correct, hopefully yes.

Still won't trigger off, unless there's a second monitor.